src/DIPS.Xamarin.UI/Internal/Xaml/ToastContainer.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+
+<ContentView xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="DIPS.Xamarin.UI.Internal.Xaml.ToastContainer"
+             x:Name="pageCodeBehind">
+    <ContentView.Content>
+        <Grid BackgroundColor="Transparent">
```

Comment:
This `Grid` can be the top element instead of a `ContentView`. Should always try to keep the number of nested views as low as possible for the app to resolve faster.